packages/scripts/tailwind.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
       './node_modules/commerce/**/*.js',
       './node_modules/react/**/*.js',
       './node_modules/convertkit/**/*.js',
+      './node_modules/quiz/**/*.js',
```

Comment:
this doesn't work for some reason and styles inside `quiz` package won't get applied. 🤔

Reply:
I think all of these would need `@skillrecordings` in the path:

`./node_modules/@skillrecordings/quiz/dist/**/*.js`

https://github.com/skillrecordings/turbo-playground/commit/8fecc68c3ea540cd9cc9946dc1a226e34e4ad4b7#diff-d6becb39d5ea8fca64b5653c55382577f5d465c73314eed7e50a92e439e6603dR7-R10

I went ahead and applied that if you want to merge it in and test